docs/src/Modal.doc.js

```diff
@@ -528,6 +528,72 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    name="Outside Event Exception Test"
```

Comment:
@emilyalwayss Would be wonderful if we could write an actual jest test for this use case (with https://github.com/testing-library/react-testing-library)

Would you be able to update the name (title) and description to something like the following:
- name: `Flyout inside of Modal Example`
-  description: `Opens a Flyout inside of a Modal and verifies that outside events work as expected with portals`

"Tests" should live in our tests and not our docs.